boards/common/iotlab/include/periph_conf_common.h

```diff
@@ -149,20 +149,6 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          ARRAY_SIZE(uart_config)
 /** @} */
 
-/**
- * @name    Real time counter configuration
- * @{
- */
-#define RTT_IRQ_PRIO        1
-
-#define RTT_DEV             RTC
-#define RTT_IRQ             RTC_IRQn
-#define RTT_ISR             isr_rtc
-#define RTT_MAX_VALUE       (0xffffffff)
-#define RTT_FREQUENCY       (1)             /* in Hz */
```

Comment:
You probably want to keep this line to retain the old behavior.

Reply:
I was actually hopping to change the default to the highest frequency as well, but I could do that in a different PR.